### PR TITLE
feat: show Advanced search name at the top of the Advanced search panel

### PR DESCRIPTION
**Pull request by PenghaiZhang**
##### Checklist

- [x] the [contributor license agreement][] is signed
- [x] commit message follows [commit guidelines][]
- [ ] tests are included
- [x] screenshots are included showing significant UI changes
- [ ] documentation is changed or added

##### Description of change

Show the name of an Advanced search at the top of the Advanced search panel.

image

### CODE REVIEW

File: react-front-end/tsrc/search/components/AdvancedSearchPanel.tsx [hunk 1]
```diff
@@ -42,6 +42,10 @@ import { languageStrings } from "../../util/langstrings";
 import { SearchPageRenderErrorContext } from "../SearchPage";
 
 export interface AdvancedSearchPanelProps {
+  /**
+   * Name of the Advanced search.
+   */
+  name?: string;
```
Comment (edalex-ian) [inline react-front-end/tsrc/search/components/AdvancedSearchPanel.tsx:48]: Sorry to be a pest, but.... `title` is probably more idiomatic. (Note how it's also the prop used for `CardHeader`.)

Could we please change?

**Review by edalex-ian — CHANGES_REQUESTED** [review]
Looks good @PenghaiZhang , but.... I think we need to change the props name.

File: react-front-end/tsrc/search/components/AdvancedSearchPanel.tsx (hunk 1 above)
Comment (PenghaiZhang) [reply react-front-end/tsrc/search/components/AdvancedSearchPanel.tsx:48]: Sure can. :smile:

**Review by edalex-ian — APPROVED** [review]
Thanks @PenghaiZhang , merge on green. :+1:

**Pull request merged**